app/controllers/api/base_controller.rb

```diff
@@ -108,5 +109,6 @@ def initialize
 
     before_action :parse_api_request, :log_api_request, :validate_api_request, :validate_api_action
     after_action :log_api_response
+    skip_before_action :validate_api_action, :only => [:options]
```

Comment:
This could be condensed to expressing the original callback (on its own line) as:

```ruby
before_action :validate_api_action, :except => [:options]
```